docs/spec/protocol.rst

```diff
@@ -371,26 +370,26 @@ The self-types in protocols follow the
   c = One()  # OK
   c = Other()  # Also OK
 
-Subtyping relationships with other types
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Assignability relationships with other types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Protocols cannot be instantiated, so there are no values whose
 runtime type is a protocol. For variables and parameters with protocol types,
-subtyping relationships are subject to the following rules:
+assignability relationships are subject to the following rules:
 
-* A protocol is never a subtype of a concrete type.
-* A concrete type ``X`` is a subtype of protocol ``P``
-  if and only if ``X`` implements all protocol members of ``P`` with
-  compatible types. In other words, subtyping with respect to a protocol is
-  always structural.
-* A protocol ``P1`` is a subtype of another protocol ``P2`` if ``P1`` defines
-  all protocol members of ``P2`` with compatible types.
+* A protocol is never assignable to a concrete type.
```

Comment:
Other than `object`?

Reply:
I think this is a good correction (a protocol type should always be assignable to object).

I'd also like to either define or replace the term "concrete type" in the spec.

But I think both of these can safely be left as a separate follow-up. The "object" change is a clear change to the meaning of the spec that isn't directly tied to this PR. "Concrete type" is more related to this PR in that it's an issue of core terminology, but it's not directly impacted by the terms or concepts introduced in this PR: the term is just as clear/unclear/accurate/inaccurate after this PR as it was before this PR. So we can handle it separately.

Filed https://github.com/python/typing/issues/1781 and https://github.com/python/typing/issues/1780